test/AutoRest.TestServer.Tests/body-complex.cs

```diff
@@ -57,6 +57,12 @@ public class BodyComplexTest: TestServerTestBase
             Assert.ThrowsAsync(Is.InstanceOf<JsonException>(), async () => await new BasicClient(ClientDiagnostics, pipeline, host).GetNotProvidedAsync());
         });
 
+        [Test]
+        public Task ThrowsIfApiVersionIsNull() => Test((host, pipeline) =>
```

Comment:
don't use `Test(..` here.

Reply:
See CheckComplexPrimitiveInteger for an example.

Reply:
But here we are trying to create an instance of `BasicClient` which takes host and pipeline.

Reply:
But here we are trying to create an instance of `BasicClient`, which takes host and pipeline. If we remove `Test(..` then how do we get host and pipeline?

Reply:
Just use some random values, you don't need an actual test server to be running and that's what `Test((host, pipeline) =>` does

Reply:
Yeah, I was using it because here we are creating an instance of `BasicClient` and it needs host and pipeline. But now I've removed `Test(..` and passed null for host and pipeline.